src/test/components/DepositWidget.test.ts

```diff
@@ -0,0 +1,11 @@
+import { getButtonText, ButtonTypes } from '../../components/DepositWidget/Row'
```

Comment:
instead of `src/test`, I think it would be better to use `test` for the test.
This way the `src` is just for source code

Reply:
Fair enough, will move it.